lib/calc/xmedian.c

```diff
@@ -44,10 +44,10 @@ static int dcmp(const void *aa, const void *bb)
 
 int f_median(int argc, const int *argt, void **args)
 {
-    static void *array;
-    static int alloc;
     int size = argc * Rast_cell_size(argt[0]);
     int i, j;
+    const int threshold = 32;
+    bool use_heap = false;
```

Comment:
Starting with #5618, we include `stdbool.h` everywhere where we use _bool_ instead of relying on transitive include through `gis.h`.